### ISSUE TITLE
preg_match() expects parameter 2 to be string, array given

### ISSUE THREAD

**Issue by pierrerigal** [erusev/parsedown#273]
Hi,

I recently updated (composer update) to latest 2.0 branch off GrahamCampbell/Laravel-Markdown on my Laravel 4.1 project and I have an issue with parsedown.

Error : 
````sh
ErrorException
preg_match() expects parameter 2 to be string, array given (ParsedownExtra.php line 252)
````

Markdown syntax with error : 
````md
###Hello world
[Link](http://google.com)
````

Markdown without error (no link) : 
````md
###Hello world
````

My composer : 
````composer
"graham-campbell/markdown": "~2.0"
````

In my composer.lock the only difference between my production (working) and my local version (not working) is parsedown version. 

Production : 1.4.5
Local : 1.5.0

Temporary fix : set parsedown to version 1.4 in composer.json

Issue firstly posted here : https://github.com/GrahamCampbell/Laravel-Markdown/issues/20

**Comment by erusev** [comment]
What's the version of Extra in your local (not working) environment?

**Comment by pierrerigal** [comment]
Sorry, I don't remember which version it was. 

But since last update, I remove the fix (parsedown 1.4) and everything is ok.

**Comment by erusev** [comment]
I'm glad to hear that.
